code/modules/mob/living/living.dm

```diff
@@ -1138,6 +1138,22 @@
 	mob_pickup(user)
 	return TRUE
 
+/mob/living/display_output(sound/S, list/sound_image, text, turf/turf_source, vol as num)
```

Comment:
You're not passing in a list anymore

Reply:
i actually am, i talked to raz and we decided to keep the list then pass it as image since mutable appearances apparently cant be made to be shown clientside only